.gitignore

```diff
@@ -4,4 +4,9 @@ html/
 */build/*
 Gemfile.lock
 */master.html
-.idea/
\ No newline at end of file
+.idea/
```

Comment:
Why have you changed the .gitignore file?
You should be using your global .gitignore file which we set up in our last git training session.

Reply:
You have closed this comment and failed to correct your mistake